nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -266,3 +266,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.nomad.extraSettingsPlugins` has been fixed to allow more than one plugin in the path.
 
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
+
+- The new option `services.caddy.enableReload` allows reloading the Caddy service instead of restarting it. Enabling this option requires the Caddy admin API to not be turned off.
```

Comment:
```suggestion
- The new option `services.caddy.enableReload` allows reloading the Caddy service instead of restarting it if a config file is the only thing that changes. Enabling this option requires the Caddy admin API to not be turned off.
```